src/marqo/core/index_management/index_management.py

```diff
@@ -58,8 +58,8 @@ class IndexManagement:
     _DEFAULT_QUERY_PROFILE_TEMPLATE = textwrap.dedent(
         '''
         <query-profile id="default">
-            <field name="maxHits">1000</field>
-            <field name="maxOffset">10000</field>
+            <field name="maxHits">{max_hits}</field>
```

Comment:
My understanding about our decision on Friday is that we'll need to hard code this config to a extremely large value so we don't get limited by Vespa during search. 
Then we add a validator to the SearchQuery to make sure `limit <= MARQO_MAX_SEARCH_LIMIT` and `offset <= MARQO_MAX_SEARCH_OFFSET`, the default value of these two env vars can be 1000 and 10000 respectively to keep the behaviour same as before. And they can be overridden as needed. 
@farshidz , @RaynorChavez , is my understanding correct?

Reply:
@papa99do, wouldn't this entail asking vespa to return a large number of results each query?

Reply:
I don't think so. We'll need to make sure the limit and offset parameter in each query does not exceed the limit set in env vars (or the default limit if env vars are not provided).